cluster/saltbase/salt/l7-gcp/glbc.manifest

```diff
@@ -43,7 +43,7 @@ spec:
     # TODO: split this out into args when we no longer need to pipe stdout to a file #6428
     - sh
     - -c
-    - '/glbc --verbose=true --default-backend-service=kube-system/default-http-backend --sync-period=600s --running-in-cluster=false --use-real-cloud=true --config-file-path=/etc/gce.conf --healthz-port=8086 1>>/var/log/glbc.log 2>&1'
+    - '/glbc --verbose=true --apiserver-host=http://localhost:8080 --default-backend-service=kube-system/default-http-backend --sync-period=600s --running-in-cluster=false --use-real-cloud=true --config-file-path=/etc/gce.conf --healthz-port=8086 1>>/var/log/glbc.log 2>&1'
```

Comment:
Correct, previously was using deprecated `clientcmd.ClusterDefaults` https://github.com/kubernetes/client-go/blob/670b68600ead2fdd5c835a9ff7239dda903640a6/tools/clientcmd/client_config.go#L49
This flag is identical to the one used for nginx ingress controller.